components/AboutPane/AboutPane.tsx

```diff
@@ -1,34 +1,11 @@
 import React, { type HTMLProps } from 'react';
-import { ExternalLink } from '../ExternalLink.js';
-import { GithubIcon } from '../Icons.js';
 import { Pane } from '../Pane.js';
 import './AboutPane.scss';
-import { version as VERSION } from '../../package.json';
 import { CommitList } from './CommitList.js';
-import GithubSponsorButton from './GithubSponsorButton.js';
 
 export default function AboutPane(props: HTMLProps<HTMLDivElement>) {
   return (
     <Pane {...props}>
-      <p>
-        {'\xA9'} npmgraph Contributors &mdash; v{VERSION}
```

Comment:
I dropped the version because it doesn't really add value here. Version 3.1.1 (December 2023) is not the same as Version 3.1.1 (today)

Also "Copyright npmgraph Contributors" is redundant and at odds with [the license file](https://github.com/npmgraph/npmgraph/blob/main/LICENSE)

Reply:
Sounds good.